### ISSUE TITLE
does pkg include devDependecies?

### ISSUE THREAD

**Issue by opensas** [zeit/pkg#729]
The question is if pkg includes the devDependencies, and if that's the case, if it's possible to exclude them from the build specifying `--only=production` or something similar

**Comment by sergiz** [comment]
It includes only what the script(s) require or you manually specify. You should read the readme thoroughly.

**Comment by opensas** [comment]
Thanks a lot, I went thru the readme, but I haven't realized that going thru the requires efectively means that dependecies in package.json were not being taken into account.
BTW, great project, it works really fine

**Comment by m4heshd** [comment]
@sergejostir Wrong. It packs a ton of unnecessary stuff. If you cared to look inside the built binary or at least ran pkg in debug mode, you'll see it packs ALL of the dependencies which most of the time a lot of them are unnecessary. Yes it excludes devDependencies. That's it.

**Comment by sergejostir** [comment]
You are responding to an answer that is more than 3 years old. This package has already seen a major version release and even changed the maintainer in the meantime.

**Comment by m4heshd** [comment]
@sergejostir I tried to use this instead of Nexe a while back. Exactly around 2018 and it had the same 2 issues that stopped me from using it. This issue and the corruption issue when a resource was changed post-build. I couldn't remember why I didn't use pkg in the first place and when I tried to use it now (because Nexe is abandoned now), ran into the same old infuriating issues again. So again.. no, you were wrong about it packing in "only what the script(s) require".
